### ISSUE TITLE
Connect to remote node

### ISSUE THREAD

**Issue by stsvilik** [parity-js/shell#61]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Parity client version**: 1.9.6
> - **Parity UI version**: 0.1.0
> - **Operating system**: MacOS
> - **Installed via**: installer
> - **Fully synchronized**: no
> - **Network**: ethereum
> - **I tried to restart both Parity UI and Parity Ethereum client**: yes

My parity node runs in a docker on a remote machine on the network. It seems that Parity UI only attempts to connect to parity node if it runs locally. Could/will you provide an option to specify remote RPC node? Please :)

Currently I use Myst UI with Parity node where Myst has `--rpc http://x.x.x.x:8545` option at launch

**Comment by stsvilik** [comment]
nvm - I need to upgrade to 0.1.1 :) - Is there a binary for Mac?

**Comment by stsvilik** [comment]
hm, still doesn't work after upgrade to 0.1.1

**Comment by Tbaut** [comment]
@stsvilik can you elaborate? What is not working?

**Comment by stsvilik** [comment]
yes, sorry. Parity UI shows up complaining that it was unable to locate parity.

**Comment by amaurymartiny** [comment]
You can currently running Mist with `--rpc http://x.x.x.x:8545`.

Can you try running Parity UI with `--ws-interface http://x.x.x.x --ws-port 8546`

**Comment by Tbaut** [comment]
This is most likely due to the flags used in the docker. Did you have a look at the port forwarding and the different interface flags [here](https://wiki.parity.io/Docker.html#open-ports)

**Comment by stsvilik** [comment]
Yes, I used most flags in docker wiki, however I did not expose 8546 - will do that next.
@amaurymartiny correct, Mist runs fine with `--rpc http://x.x.x.x:8545`. I thought Parity UI would re-use same port, but it sounds like it needs `8546`.

**Comment by stsvilik** [comment]
Would be great for future versions to prompt user to enter remote IP/Port when local parity instance is not found. :)
